activerecord/test/cases/relations_test.rb

```diff
@@ -256,6 +256,18 @@ def test_finding_with_subquery_with_eager_loading_in_from
     assert_equal relation.to_a, Comment.select("a.*").from(relation, :a).to_a
   end
 
+  unless current_adapter?(:SQLite3Adapter)
```

Comment:
Using unions currently produces invalid queries for sqlite - it wraps them in extra parentheses.